yambo-front/src/components/CardCreate.tsx

```diff
@@ -288,12 +283,11 @@ const CardCreate: FC = () => {
           tabIndex={1}
           name="kanji"
           type="text"
-          placeholder="素晴らしい"
+          placeholder={cardPlaceholder}
           value={kanji}
           disabled={hiragana ? true : false}
           onChange={handleChange}
-          onClick={(e: MouseEvent) => setCardPlaceholder("")}
-          onKeyDown={handleKeyDown}></FormInput>
+          onClick={() => setCardPlaceholder("")}></FormInput>
```

Comment:
Since this is undefined, you may need to add 'undefined' (without quotes) to the typing (e.g. <string | undefined>)